src/pyOpenMS/pxds/JavaInfo.pxd

```diff
@@ -6,7 +6,15 @@ cdef extern from "<OpenMS/SYSTEM/JavaInfo.h>" namespace "OpenMS":
 
     cdef cppclass JavaInfo:
 
-        JavaInfo() nogil except +
+        JavaInfo() nogil except + # wrap-doc:Detect Java and retrieve information
         JavaInfo(JavaInfo &) nogil except +
 
         bool canRun(String java_executable) nogil except +
+            # wrap-doc:
+                #   Determine if Java is installed and reachable
+                #   -----
+                #   The call fails if either Java is not installed or if a relative location is given and Java is not on the search PATH
+                #   -----
+                #   :param java_executable: Path to Java executable. Can be absolute, relative or just a filename
+                #   :param verbose: On error, should an error message be printed to OPENMS_LOG_ERROR?
```

Comment:
```suggestion
```

Reply:
The parameter `verbose` from the original function (see below) is not wrapped and always true in terms of pyopenms - so this line should be removed. 

`static bool canRun(const String& java_executable, bool verbose_on_error = true); `